mmdet/models/losses/multipos_cross_entropy_loss.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Optional
+
+import torch
+from mmengine.model import BaseModule
+from torch import Tensor
+
+from mmdet.registry import MODELS
+from .utils import weight_reduce_loss
+
+
+@MODELS.register_module()
+class MultiPosCrossEntropyLoss(BaseModule):
+    """multi-positive targets cross entropy loss."""
+
+    def __init__(self, reduction: str = 'mean', loss_weight: float = 1.0):
+        super(MultiPosCrossEntropyLoss, self).__init__()
+        self.reduction = reduction
+        self.loss_weight = loss_weight
+
+    def multi_pos_cross_entropy(self,
+                                pred: Tensor,
+                                label: Tensor,
+                                weight: Optional[Tensor] = None,
+                                reduction: str = 'mean',
+                                avg_factor: Optional[float] = None) -> Tensor:
+        """
+        Args:
+            pred (torch.Tensor): The prediction.
+            label (torch.Tensor): The assigned label of the prediction.
+            weight (torch.Tensor): The element-wise weight.
+            reduction (str): Same as built-in losses of PyTorch.
+            avg_factor (float): Average factor when computing
+                the mean of losses.
+        Returns:
+            torch.Tensor: Calculated loss
+        """
+
+        pos_inds = (label >= 1)
```

Comment:
what is the dtype of label? long or float?

Reply:
long